### ISSUE TITLE
lqosd segfault on start

### ISSUE THREAD

**Issue by rchac** [LibreQoE/LibreQoS#606]
```
     Loaded: loaded (/etc/systemd/system/lqosd.service; enabled; vendor preset: enabled)
     Active: failed (Result: signal) since Sun 2024-12-15 21:15:46 UTC; 1min 40s ago
    Process: 1494 ExecStart=/opt/libreqos/src/bin/lqosd (code=killed, signal=SEGV)
   Main PID: 1494 (code=killed, signal=SEGV)
```

**Comment by rchac** [comment]
Appears to be on .deb builds only

**Comment by rchac** [comment]
Fixed in v1.5-beta-5
